examples/platforms/da15000/crypto/da15000-mbedtls-config.h

```diff
@@ -0,0 +1,2632 @@
+/**
+ * \file config.h
+ *
+ * \brief Configuration options (set of defines)
+ *
+ *  This set of compile-time options may be used to enable
+ *  or disable features selectively, and reduce the global
+ *  memory footprint.
+ *
+ *  Copyright (C) 2006-2015, ARM Limited, All Rights Reserved
```

Comment:
All code that does not have "The OpenThread Authors" copyright must live in `third_party`.